clonelists/Sony - PlayStation 2.json

```diff
@@ -2506,10 +2577,12 @@
 		],
 		"Shin Megami Tensei - Persona 3": [
 			["Shin Megami Tensei - Persona 3 FES", 0],
-			["Persona 3 FES", 0],
 			["P3 - Persona 3", 1],
 			["Persona 3", 1]
 		],
+		"Shin Megami Tensei - Persona 3 FES": [
```

Comment:
To my understanding the previous entry was correct (zero priority titles are listed under a single title in order of priority, they aren't isolated as a separate entry for matching with their clones -- although there will be legacy entries where this old way of formatting things is in effect). Are you seeing unexpected behavior here?

A heads up that this way of formatting also likely won't be supported in Retool 2.0.

Reply:
I've reverted this in a new commit so it works as intended.